root/etc/systemd/system/netdata.service.d/cleanup.conf

```diff
@@ -1,2 +1,5 @@
 [Service]
 ExecStartPre=-/usr/bin/find /var/cache/netdata/ -type f -empty -delete
+ReadWriteDirectories=/etc/netdata/
```

Comment:
What is the effect of this directive? How does it alter the service permissions on that directory?

I cannot understand it even after reading the systemd.exec manpage:

> Sets up a new file system namespace for executed processes. These options may be used to limit access a process might have to the main file system hierarchy.

    $ ls -ld /etc/netdata/
    drwxr-xr-x 7 root root 129 Jun 17 16:12 /etc/netdata/

Reply:
The process is started inside a namespace which have read-only access to the filesystem.

See the commit message for more info: https://github.com/NethServer/nethserver-netdata/pull/4/commits

Reply:
Ok I guess read-only is triggered by

    ProtectSystem=full

From the manpage:



> ProtectSystem=
>           Takes a boolean argument or "full". If true, mounts the /usr and /boot directories read-only for processes invoked by this unit. If set to "full", the /etc directory is mounted read-only, too.

Reply:
Yes, it's that one!